.github/workflows/release_docker.yml

```diff
@@ -27,47 +28,102 @@ on:
         default: false
 
 jobs:
-  build-iota-node:
-    if: github.event_name == 'workflow_dispatch' && github.event.inputs.iota_node == 'true' || github.event_name == 'release'
-    runs-on: self-hosted
+  prepare:
+    outputs:
+      images: ${{ steps.images.outputs.images }}
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Define images to build
+        id: images
+        run: |
+          images_array=()
+
+          # Check iota_node: true if the input is set to true and the event is workflow_dispatch,
+          # or if the event is a release (in which case the job should run regardless of the input).
+          if [ "${{ github.event_name == 'workflow_dispatch' && github.event.inputs.iota_node == 'true' || github.event_name == 'release' }}" == "true" ]; then
```

Comment:
Just curious, do we actually need the final `== "true"`? Isn't it already a boolean?

Reply:
Because bash is pain and this is safer.